src/collective/documentgenerator/config.py

```diff
@@ -7,10 +7,16 @@
 
 from plone import api
 
-POD_FORMATS = (('doc', 'Microsoft Word'),
-               ('odt', 'Open Document Format (text)'),
-               ('rtf', 'Rich Text Format (RTF)'),
-               ('pdf', 'Adobe PDF'))
+ODS_FORMATS = [('ods', 'Open Document Format (tab)'),
```

Comment:
Hi @DieKatze @boulch 

I would add the file extension to the Text like 'Microsoft Word (.doc)' and rename 'Open Document Format' to 'LibreOffice Calc (.ods), LibreOffice Writer (.odt)' so it is clear for endusers, I wondered myself what those notations where about especially (tab) and (text) so...